src/aggregates.rs

```diff
@@ -171,7 +157,7 @@ impl AggregateSignature {
             if msg[i].len() != MSG_LENGTH {
```

Comment:
Do we want to restrict the message length to 32 bytes in `AggregateVerify`?

Although all messages in all the use cases in the spec are 32 bytes.

Reply:
It's an interesting call on this, on one hand the BLS standard message length is unrestricted,
however as you mention all calls from Eth2.0 should have 32 byte messages.

I think it may be more appropriate to put the message length check inside lighthouse itself [here](https://github.com/sigp/lighthouse/blob/master/eth2/utils/bls/src/signature.rs#L28), [here](https://github.com/sigp/lighthouse/blob/master/eth2/utils/bls/src/aggregate_signature.rs#L50) and [here](https://github.com/sigp/lighthouse/blob/master/eth2/utils/bls/src/aggregate_signature.rs#L62).
Then keep `milagro_bls` conforming to the BLS standard such that it is available for other users.